packages/x-date-pickers-pro/src/internals/models/rangePickerProps.ts

```diff
@@ -15,4 +15,8 @@ export interface BaseRangeNonStaticPickerProps {
    * Ignored if the field has several inputs.
    */
   inputRef?: React.Ref<HTMLInputElement>;
+  /**
+   * Name attribute used by the `input` element in the Field.
```

Comment:
On this one the last line made sense since range pickers can have several inputs

```suggestion
   * Name attribute used by the `input` element in the Field.
   * Ignored if the field has several inputs.
```

Reply:
And you did not forward the `name` prop to the field in `useDesktopRangePicker` and `useMobileRangePicker`
You can add it with `inputRef` in both those files (so only if `fieldType === 'single-input'`).

And after that I think we are good to go :tada:

Reply:
Ah.. right. Good catch 👍